martin-tile-utils/src/lib.rs

```diff
@@ -248,13 +251,31 @@ pub fn webmercator_to_wgs84(x: f64, y: f64) -> (f64, f64) {
     (lng, lat)
 }
 
+/// transform WGS84 to WebMercator
+// from https://github.com/Esri/arcgis-osm-editor/blob/e4b9905c264aa22f8eeb657efd52b12cdebea69a/src/OSMWeb10_1/Utils/WebMercator.cs
```

Comment:
It's from esri.

Reply:
I think this code is actually from the OSM wiki that has all this math